landing-page/content/common/releases/release-notes.md

```diff
@@ -23,16 +23,12 @@ url: releases
 ## Downloads
 
 The latest version of Iceberg is [{{% icebergVersion %}}](https://github.com/apache/iceberg/releases/tag/apache-iceberg-{{% icebergVersion %}}).
+You can download the [source code](https://www.apache.org/dyn/closer.cgi/iceberg/apache-iceberg-{{% icebergVersion %}}/apache-iceberg-{{% icebergVersion %}}.tar.gz) 
+and verify its [signature](https://downloads.apache.org/iceberg/apache-iceberg-{{% icebergVersion %}}/apache-iceberg-{{% icebergVersion %}}.tar.gz.asc) 
+and [SHA512 checksum](https://downloads.apache.org/iceberg/apache-iceberg-{{% icebergVersion %}}/apache-iceberg-{{% icebergVersion %}}.tar.gz.sha512).
```

Comment:
I think it's better to have a `Multi-engine Support` page in the landing page (and as a highlight feature at home page), instead of tracking all the runtime jars here. Any thoughts?